packages/ui/src/components/StoryCard/StoryCard.tsx

```diff
@@ -110,3 +100,20 @@ export function StoryCard({
         </div>
     );
 }
+
+export namespace StoryCard {
+    export type DisplayedCategory = CategoriesList.DisplayedCategory;
+
+    export interface Props {
+        story: StoryWithImage | AlgoliaStory;
```

Comment:
Let's define a local type for this: `DisplayedStory` and make it require only the props we really need in this component: `{ title, subtitle, href }`.